components/director/internal/domain/tenant/fixtures_test.go

```diff
@@ -0,0 +1,124 @@
+package tenant_test
+
+import (
+	"database/sql/driver"
+	"errors"
+
+	"github.com/kyma-incubator/compass/components/director/internal/domain/tenant"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/kyma-incubator/compass/components/director/pkg/pagination"
+)
+
+const (
+	testExternal = "external"
+	testID       = "foo"
+	testName     = "bar"
+	testPageSize = 3
+	testCursor   = ""
+	testProvider = "Compass"
+)
+
+var (
+	testError        = errors.New("test error")
+	testDescription  = "bazz"
+	testTableColumns = []string{"id", "name", "external_tenant", "internal_tenant", "provider_name", "status"}
+)
+
+func fixModelTenantMapping(id, name string) *model.TenantMapping {
+	return &model.TenantMapping{
+		ID:             id,
+		Name:           name,
+		ExternalTenant: testExternal,
+		InternalTenant: testID,
+		Provider:       testProvider,
+		Status:         model.Active,
+	}
+}
+
+func fixModelTenantMappingWithExternalTenant(id, name, externalTenant string) *model.TenantMapping {
+	tenant := fixModelTenantMapping(id, name)
+	tenant.ExternalTenant = externalTenant
+	return tenant
+}
+
+func fixInactiveModelTenantMapping(id, name string) *model.TenantMapping {
```

Comment:
I am sick and tired of our fixtures :)
What do you think about the following approach:
```
func NewTenantMapping(id, name string) *TenantMapping {
	return &TenantMapping{ID: id, Name: name}
}

func (t TenantMapping) WithProvider(provider string) *TenantMapping {
	t.Provider = provider
	return &t
}
```

then, in tests we can write tm := NewTenantMapping("aaa", "bbb").WithProvider("ccc")